IronFoundry.Container.Shared/Utilities/ProcessHelper.cs

```diff
@@ -103,13 +103,33 @@ public void Dispose()
 
             public void Kill()
             {
+                // FROM: https://msdn.microsoft.com/en-us/library/windows/desktop/ms681382%28v=vs.85%29.aspx
+                const int accessDeniedErrorCode = 0x5;
+
                 if (process.HasExited) return;
-                process.Kill();
+                try
+                {
+                    process.Kill();
+                }
+                catch (Win32Exception ex)
+                {
+                    // Access Denied can be thrown if we attempt to kill a process while it is exiting.
+                    // See notes at: https://msdn.microsoft.com/en-us/library/system.diagnostics.process.kill%28v=vs.110%29.aspx
+                    if (ex.ErrorCode != accessDeniedErrorCode)
+                    {
+                        throw;
+                    }
+                }
             }
 
             public void RequestExit()
             {
+                // MO: We can't do this anymore.  RequestForExit as currently implemented sends
+                // uses GenerateConsoleCtrlEvent which wil send the request to the whole console
+                // group.  At the moment that console group includes the ContainerHost.
+                //
                 //ProcessHelper.SendSignal(process.Id, false);
+                Kill();
```

Comment:
The existing callers to `RequestExit()` would basically timeout and then kill the process.

I think we should get rid of `RequestExit()` and replace it with `Stop(int timeout)` using the same logic as the `StopProcessHandler`.  Right now we'll just have to kill the process, but it's a better interface for what we want to do and we can contain the logic in a single place.